api/MutationObserver.json

```diff
@@ -0,0 +1,330 @@
+              "version_added": "18",
+              "version_removed": "26",
+              "prefix": "Webkit"
+            }
+          ],
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "14"
+          },
+          "firefox_android": {
+            "version_added": "14"
+          },
+          "ie": {
+            "version_added": "11"
+          },
+          "opera": {
+            "version_added": "15"
+          },
+          "opera_android": {
+            "version_added": "15"
+          },
+          "safari": [
+            {
+              "version_added": "7"
+            },
+            {
+              "version_added": "6",
+              "version_removed": "7",
+              "prefix": "Webkit"
+            }
+          ],
+          "safari_ios": [
+            {
+              "version_added": "7"
+            },
+            {
+              "version_added": "6",
+              "version_removed": "7",
+              "prefix": "Webkit"
+            }
+          ],
+          "samsunginternet_android": {
+            "version_added": true
+          },
+          "webview_android": [
+            {
+              "version_added": "26"
+            },
+            {
+              "version_added": "18",
+              "version_removed": "26",
+              "prefix": "Webkit"
+            }
+          ]
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "MutationObserver": {
+        "__compat": {
+          "description": "<code>MutationObserver()</code> constructor",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MutationObserver/MutationObserver",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "26"
+              },
+              {
+                "version_added": "18",
+                "version_removed": "26",
+                "prefix": "Webkit"
+              }
+            ],
+            "chrome_android": [
+              {
+                "version_added": "26"
+              },
+              {
+                "version_added": "18",
+                "version_removed": "26",
+                "prefix": "Webkit"
+              }
+            ],
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "14"
+            },
+            "firefox_android": {
+              "version_added": "14"
+            },
+            "ie": {
+              "version_added": "11"
+            },
+            "opera": {
+              "version_added": "15"
+            },
+            "opera_android": {
+              "version_added": "15"
```

Comment:
Opera Android began using WebKit/Blink with version `14`, 1 version before Opera for Desktop.